README.md

```diff
@@ -11,7 +11,7 @@ https://github.com/AllenInstitute/sonata/blob/master/docs/SONATA_DEVELOPER_GUIDE
 ## Building the C++ library
 
 ```shell
-git clone git@github.com:BlueBrain/libsonatareport.git --recursive
+git clone --recursive git@github.com:openbraininstitute/libsonatareport.git
```

Comment:
Is there any other occurrence in the repo? We could replace them all in this PR.

Reply:
done